Firebase/Auth/Source/FIRAuth.m

```diff
@@ -1137,6 +1137,39 @@ - (void)sendSignInLinkToEmail:(nonnull NSString *)email
   });
 }
 
+- (void)updateCurrentUser:(FIRUser *)user completion:(nullable FIRUserUpdateCallback)completion {
+  void (^updateUserBlock)(FIRUser *user) = ^(FIRUser *user) {
+    NSError *error;
+    [self updateCurrentUser:user byForce:YES savingToDisk:YES error:(&error)];
+    if (error) {
+      if (completion) {
+        completion(error);
+      }
+      return;
+    }
+    completion(nil);
```

Comment:
This is missing a nullability check for `completion`.

Reply:
Done, thanks.